swift/Tests/Blockchains/NebulasTests.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright © 2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import XCTest
+import TrustWalletCore
+
+class NebulasTests: XCTestCase {
+
+    func testAddressFromPublicKey() {
+        let privateKey = PrivateKey(data: Data(hexString: "d2fd0ec9f6268fc8d1f563e3e976436936708bdf0dc60c66f35890f5967a8d2b")!)!
+        let publicKey = privateKey.getPublicKeySecp256k1(compressed: false)
+        let address = NebulasAddress(publicKey: publicKey)
+
+        XCTAssertEqual(address.description, "n1V5bB2tbaM3FUiL4eRwpBLgEredS5C2wLY")
+    }
+
+    func testSigner() {
+        let arr: [Int8] = [10, 6, 98, 105, 110, 97, 114, 121]
+        let input = NebulasSigningInput.with {
+            $0.chainID = Data(hexString: "01")!
+            $0.fromAddress = "n1V5bB2tbaM3FUiL4eRwpBLgEredS5C2wLY"
+            $0.nonce = Data(hexString: "07")!
+            $0.gasPrice = Data(hexString: "0f4240")!     //1000000
+            $0.gasLimit = Data(hexString: "030d40")!     //200000
+            $0.toAddress = "n1SAeQRVn33bamxN4ehWUT7JGdxipwn8b17"
+            $0.amount = Data(hexString: "98a7d9b8314c0000")!  //11000000000000000000ULL
+            $0.timestamp = Data(hexString: "5cfc84ca")!   //1560052938
+            $0.payload = Data(bytes: arr, count: 8)
```

Comment:
I think you can just use `Data(hexString:)`?

Reply:
Let me try.

Reply:
Can I convert the payload from a string? that's more understandably.